src/CMakeLists.txt

```diff
@@ -606,7 +608,7 @@ set(headers
 
 if (CGNS_ENABLE_FORTRAN)
   list(APPEND headers
-    ${CMAKE_CURRENT_BINARY_DIR}/cgns.mod)
+    $<TARGET_FILE_DIR:cgns_static>/cgns.mod)
```

Comment:
This change should be safe on all platforms, with all configurations. For multi-config generators, like MSVS, targets get put into a subdirectory with the `$<CONFIG>` name. By piggybacking off of the location of the `cgns_static` directory we can determine where the compiler put the .mod file, assuming, users aren't doing anything crazy like setting [`CMAKE_Fortran_MODULE_DIRECTORY`](https://cmake.org/cmake/help/v3.13/variable/CMAKE_Fortran_MODULE_DIRECTORY.html).

If you like, I could add a check for this, and if set, then use that location instead. This would be fairly pathological on the part of the user, but you never know.

Reply:
We should probably allow for user's changes.